dask/array/rechunk.py

```diff
@@ -662,3 +691,86 @@ def format_plan(plan):
     [(3*[10], 2*[15]), ([30], 3*[10])]
     """
     return [format_chunks(c) for c in plan]
+
+
+def _get_chunks(n, chunksize):
+    leftover = n % chunksize
+    n_chunks = n // chunksize
+
+    chunks = [chunksize] * n_chunks
+    if leftover:
+        chunks.append(leftover)
+    return tuple(chunks)
+
+
+def _even_chunksize(N: int, n_chunks: Union[int, float]) -> int:
+    """
+    Find a chunk size that splits an array into even sized chunks.
+
+    Parameters
+    ----------
+    N : int
+        Length of the array
+    n_chunks : int, float.
+        Approximate number of chunks for the array
+
+    Returns
+    -------
+    chunksize : int
+        The chunksize for the array. When an array is chunked with this
+        chunksize,
+
+        * The length of every chunk will be approximately the same.
+        * The number of chunks will be approximately ``n_chunks`` (within one).
+
+    Examples
+    --------
+    >>> import numpy as np
+    >>> import dask.array as da
```

Comment:
I don't think this works as written. You are missing an import:

```python
from dask.array.rechunk import _even_chunksize
```
Also I get a different result: 

```python
((315,
  315,
  315,
  315,
  315,
  315,
  315,
  315,
  315,
  315,
  315,
  315,
  315,
  315,
  315,
  305),)
```